packages/cashier/src/Stores/__tests__/iframe-store.spec.js

```diff
@@ -0,0 +1,129 @@
+import IframeStore from '../iframe-store';
+
+let iframe_store, root_store, WS;
+
+beforeEach(() => {
+    WS = {};
+    root_store = {
+        client: {
+            setVerificationCode: jest.fn(),
+        },
+        modules: {
+            cashier: {
+                general_store: {
+                    active_container: 'payment_agent',
+                    setLoading: jest.fn(),
+                },
+            },
+        },
+        ui: {
+            is_dark_mode_on: true,
+            is_mobile: false,
+        },
+    };
+    iframe_store = new IframeStore({ WS, root_store });
+});
+
+describe('IframeStore', () => {
+    it('should set session_timeout and invoke removeOnIframeLoaded function', () => {
+        const spyRemoveOnIframeLoaded = jest.spyOn(iframe_store, 'removeOnIframeLoaded');
+        iframe_store.setSessionTimeout(true);
+
+        expect(iframe_store.is_session_timeout).toBeTrue();
+        expect(spyRemoveOnIframeLoaded).toHaveBeenCalledTimes(1);
+    });
+
+    it('should remove onIframeLoaded function', () => {
+        const removeEventListener = jest.spyOn(window, 'removeEventListener');
+        iframe_store.onIframeLoaded = jest.fn();
+        iframe_store.removeOnIframeLoaded();
+
+        expect(removeEventListener).toHaveBeenCalledWith('message', expect.any(Function), false);
+        expect(iframe_store.onIframeLoaded).toBe('');
+    });
+
+    it('should clear timeout cashier url', () => {
+        jest.useFakeTimers();
+
+        iframe_store.timeout_session = 999;
+        iframe_store.clearTimeoutCashierUrl();
+
+        expect(clearTimeout).toHaveBeenCalledWith(999);
+        jest.useRealTimers();
+    });
+
+    it('should set timeout cashier url', () => {
+        jest.useFakeTimers();
+
+        const spyClearTimeoutCashierUrl = spyOn(iframe_store, 'clearTimeoutCashierUrl');
+        const spySetSessionTimeout = spyOn(iframe_store, 'setSessionTimeout');
```

Comment:
Shouldn't it be `jest.spyOn`?

Reply:
@nijil-binary oh, yes, thx for noticed, will fix